components/loki.libsonnet

```diff
@@ -691,12 +694,13 @@ local k = (import 'ksonnet/ksonnet.beta.4/k.libsonnet');
   manifests+:: {
     'config-map': loki.configmap,
   } + {
-    [normalizedName(name) + '-deployment']:
-      if name == 'ingester' then
-        newStatefulSet(name, loki.components[name])
-      else
-        newDeployment(name, loki.components[name])
+    [normalizedName(name) + '-deployment']: newDeployment(name, loki.components[name])
+    for name in std.objectFields(loki.components)
+    if !isStatefulSet(name)
+  } + {
+    [normalizedName(name) + '-statefulset']: newDeployment(name, loki.components[name])
```

Comment:
this should have `newStatefulSet` I think

Reply:
:facepalm: naming is hard, because of renaming ;D